src/pages/Editor/components/Slider/NumberSlider.jsx

```diff
@@ -1,9 +1,28 @@
 import styles from "./slider.module.scss";
 import React, { useState } from "react";
+import { Tooltip, Avatar } from "@material-ui/core";
+import { withStyles, makeStyles } from "@material-ui/core";
+
+const CustomTooltip = withStyles({
```

Comment:
this should be defined in CSS.

Reply:
Hey, @anushbhatia, these are all material imports, thus, they are imported within the component jsx. Check below for the usage.
https://material-ui.com/getting-started/usage/(url)